src/models/tests/objectiveStatus.test.js

```diff
@@ -0,0 +1,378 @@
+      });
+      grantThree = await Grant.findOne({ where: { id: 476469 } });
+
+      await Grant.create({
+        ...mockGrant,
+        id: 476470,
+        number: '58947D84',
+        recipientId: recipientFour.id,
+        programSpecialistName: user.name,
+        programSpecialistEmail: user.email,
+      });
+      grantFour = await Grant.findOne({ where: { id: 476470 } });
+
+      // Reports.
+      reportOne = await ActivityReport.create({ ...sampleReport, endDate: '2022-09-30T12:00:00Z' });
+      reportTwo = await ActivityReport.create({
+        ...sampleReport,
+        submissionStatus: REPORT_STATUSES.SUBMITTED,
+        calculatedStatus: REPORT_STATUSES.APPROVED,
+        endDate: '2022-09-29T12:00:00Z',
+      });
+      reportThree = await ActivityReport.create({ ...sampleReport });
+
+      reportOnlyUsingObjective = await ActivityReport.create({ ...sampleReport, endDate: '2022-09-30T12:00:00Z' });
+
+      // Activity Recipients.
+      await ActivityRecipient.create(
+        { activityReportId: reportOne.id, grantId: grantOne.id },
+      );
+      await ActivityRecipient.create(
+        { activityReportId: reportTwo.id, grantId: grantTwo.id },
+      );
+      await ActivityRecipient.create(
+        { activityReportId: reportThree.id, grantId: grantThree.id },
+      );
+      await ActivityRecipient.create(
+        { activityReportId: reportOnlyUsingObjective.id, grantId: grantFour.id },
+      );
+      // Goals.
+      goal = await Goal.create(
+        {
+          name: 'goal 1',
+          grantId: grantOne.id,
+        },
+      );
+      goalTwo = await Goal.create(
+        {
+          name: 'goal 2',
+          grantId: grantTwo.id,
+        },
+      );
+
+      // Objectives.
+      objective = await Objective.create(
+        {
+          title: 'Objective Used on Reports',
+          goalId: goal.id,
+          status: 'Not Started',
+        },
+      );
+      objectiveTwo = await Objective.create(
+        {
+          title: 'Objective Used on Reports Two',
+          goalId: goalTwo.id,
+          status: 'Not Started',
+        },
+      );
+      objectiveTwoB = await Objective.create(
+        {
+          title: 'Objective Used on Reports Two B',
+          goalId: goalTwo.id,
+          status: 'In Progress',
+        },
+      );
+
+      // ARO's
+      ActivityReportObjective.create({
+        activityReportId: reportOne.id,
+        objectiveId: objective.id,
+        status: 'Completed',
+      });
+
+      ActivityReportObjective.create({
+        activityReportId: reportTwo.id,
+        objectiveId: objective.id,
+        status: 'In Progress',
+      });
+
+      ActivityReportObjective.create({
+        activityReportId: reportThree.id,
+        objectiveId: objective.id,
+        status: 'Suspended',
+      });
+
+      ActivityReportObjective.create({
+        activityReportId: reportOnlyUsingObjective.id,
+        objectiveId: objectiveTwo.id,
+        status: 'Completed',
```

Comment:
```suggestion
        status: 'Complete',
```

Reply:
I think you are confusing PR's. 

This change was done in: https://github.com/HHS/Head-Start-TTADP/pull/1048